NewHorizons/Utility/AstroObjectLocator.cs

```diff
@@ -142,6 +152,12 @@ public static GameObject[] GetChildren(AstroObject primary)
                     otherChildren.Add(SearchUtilities.Find("DB_SmallNest_Body"));
                     otherChildren.Add(SearchUtilities.Find("DB_Elsinore_Body"));
                     break;
+                case AstroObject.Name.SunStation:
```

Comment:
Doesn't the sun station not use the SunStation enum

Reply:
Yes but I change it in a patch